pkg/containertools/dockerfilegenerator.go

```diff
@@ -8,14 +8,15 @@ import (
 )
 
 const (
-	defaultBinarySourceImage = "quay.io/operator-framework/upstream-opm-builder"
-	DefaultDbLocation        = "/database/index.db"
-	DbLocationLabel          = "operators.operatorframework.io.index.database.v1"
+	defaultBinarySourceImage    = "quay.io/operator-framework/upstream-opm-builder"
```

Comment:
Is there any reason we can't export this default const variable like we did with the others?

Reply:
@timflannagan this is a generated file 
```
go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . DockerfileGenerator
```